Додає стилі контактів та оновлює розмітку

biochipman opened the pull request:
- Додає інтерактивну мапу google з адаптацією під різні екрани
- Додає стилі секції контактів

biochipman committed:
Додає стилі контактів та оновлює розмітку

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1,75 +1,71 @@
+          </li>
+
+          <li class="mimino-address__item">
+            <h4 class="mimino-address__title visually-hidden">Адреса</h4>
```
<h4 class="mimino-address__title visually-hidden">Адреса</h4>
О, класне рішення!

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  fill: $color-text-tips;
+}
+
+.contacts__map {
```
Цей клас .contacts__map стоїть в розмітці нижче, ніж .map-responsive. Перестав у стилях після  .map-responsive.

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+.contacts__map {
+  width: 100%;
+  box-shadow: 5px 5px 70px 17px rgba(0, 0, 0, 0.25);
+  border-radius: 2px;
```
Ці правила перенеси на .map-responsive 
  box-shadow: 5px 5px 70px 17px rgba(0, 0, 0, 0.25);
  border-radius: 2px;

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  @include desktop {
+    color: $color-logo-gray;
+  }
+}
```
1. Класи прописуються по мірі їх згадування у розмітці.
2. Клас logo - загальний, на нього не можна вішати інші правила, для цього в тебе є contacts__logo (переназви клас, щоб не було повторів і конфліктів), на нього кинь тільки колір, розмір шрифта для різних пристроїв заданий для logo

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1,75 +1,71 @@
-      </ul>
-    </address>
+    <div class="mimino-container">
+      <p class="logo section-logo">Mimiho</p>
```
section-logo заміни на contacts__logo

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  @media screen and (max-width: 1023px) {
+    display: flex;
+    padding: 56px 0 40px 0;
+  }
```
Прибери flex з секції
 
 .contacts {
  padding: 36px 0 32px 0;
  background-color: $background-main-light;
  
  @include tablet {
    padding: 56px 0 40px 0;
  }

  @include desktop {
    padding: 56px 0 73px 0;
  }
}

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+.contacts__content {
+  @include tablet {
+    display: flex;
+    align-items: flex-start;
```
align-items: flex-start; - не змінює положення елементів, можна прибрати

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  @include desktop {
+    margin-right: 108px;
+  }
+}
```
В такому разі margin-bottom: 32px; залишається на всіх екранах і розширює секцію на таблетці і десктопі. 
Заміни на:

.mimino-container {
  @media screen and (max-width: 1023px) {
    margin-bottom: 32px;
  }

  @include tablet {
    margin-right: 100px;
  }

  @include desktop {
    margin-right: 108px;
  }
}

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+    margin-right: 108px;
+  }
+}
+
```
Тут .contacts__logo {
  color: $color-logo-dark;

  @include desktop {
    color: $color-logo-gray;
  }
}

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+
+.contacts__title {
+  margin-top: 16px;
+  margin-bottom: 40px;
```
margin-top: 16px;
  margin-bottom: 40px; 
  
  Змінюються в залежності від пристрою, а не фіксовані

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  font-size: 16px;
+  font-weight: 600;
+  line-height: calc(20 / 16);
+  letter-spacing: 0.02em;
```
Застосуй міксин   
font-size: 16px;
  font-weight: 600;
  line-height: calc(20 / 16);
  letter-spacing: 0.02em;

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  @include desktop {
+    font-size: 20px;
+    line-height: calc(24 / 20);
+  }
```
Застосуй міксин, замість повторюваних значень пропиши null
    font-size: 20px;
    line-height: calc(24 / 20);

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  }
+}
+
+.mimino-address__title {
```
.mimino-address__title  не на своєму місці. Класи пишуться по порядку згадування, а не важливості контенту
mimino-address__list
mimino-address__item 
mimino-address__item--restaraunt
mimino-address__title
mimino-address__link
mimino-address__icon і так далі

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  font-weight: 500;
+  font-style: normal;
+  line-height: calc(24 / 20);
+  letter-spacing: 0.02em;
```
Застосуй міксин
  font-size: 20px;
  font-weight: 500;
  font-style: normal;
  line-height: calc(24 / 20);
  letter-spacing: 0.02em;

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  }
+}
+
+.mimino-address__item--restaraunt {
```
Замість додаткових класів-ковбасок можна використати це, тільки в розмітці прибери
 
.mimino-address__item {
  @media screen and (max-width: 1023px) {
    display: flex;
    align-items: center;
  }
}

.mimino-address__item:not(:last-child) {
  margin-bottom: 16px;
}

.mimino-address__item:nth-child(1) {
  @include tablet {
    margin-bottom: 24px;
  }
}

.mimino-address__item:nth-child(2) {
  @include tablet {
    margin-bottom: 48px;
  }
}

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1,75 +1,71 @@
+      <h3 class="contacts__title">Ресторанно-готельний комплекс</h3>
+      <address class="mimino-address">
+        <ul class="mimino-address__list list">
+          <li class="mimino-address__item mimino-address__item--restaraunt">
```
Цю ковбаску можна прибрати mimino-address__item--restaraunt і інші також, в стилях приписала, як можна замінити

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+.mimino-address__link {
+  display: inline-flex;
+  align-items: center;
+  padding: 7px 8px 7px 8px;
```
padding: 8px; 
Там іконка з'їхала

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  font-weight: 400;
+  font-style: normal;
+  line-height: calc(17 / 14);
+  letter-spacing: 0.02em;
```
Заміни на міксин 
  font-size: 14px;
  font-weight: 400;
  font-style: normal;
  line-height: calc(17 / 14);
  letter-spacing: 0.02em;

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  display: inline-flex;
+  align-items: center;
+  padding: 7px 8px 7px 8px;
+  text-decoration: none;
```
Прибери text-decoration: none;
Для цього тоїть клас link

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1,75 +1,71 @@
+            <div class="tel-container">
+              <a
+                href="tel:+380668491089"
+                class="mimino-address__link mimino-address__link--br link"
```
mimino-address__link--br 
Це можна прибрати і використати mimino-address__link + mimino-address__link

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+  }
+}
+
+.mimino-address__link--br {
```
mimino-address__link--br 
Це можна прибрати і використати mimino-address__link + mimino-address__link

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -1,75 +1,71 @@
+          <li class="mimino-address__item">
+            <h4 class="mimino-address__title visually-hidden">Адреса</h4>
+            <a
+              class="mimino-address__link mimino-address__link--address"
```
Загубив клас link

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -0,0 +1,195 @@
+}
+
+.map-responsive {
+  margin: 0 auto;
```
На макеті каринка прижата до падінгу секції знизу. Я б тут ще поставила align-self: flex-end;

kuhai-anna reviewed (changes requested):
Чекаю на зміни і адаптацію під екран 320рх

biochipman committed:
Фіксить розмітку та стилі секції контактів

- Змінена інтерактивна карта
- Прибрані зайві класи
- Шрифти підключені через міксин
- Додана адаптація під екрани 320px

biochipman committed:
Фіксить розмір інтерактивної мапи

Адаптація мапи під екран 320px

biochipman committed:
Змінює стилі інтерактивної карти

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -196,20 +195,5 @@
 }
 
 .contacts__map {
```
Заміни на .contacts__map {
    width: 100%;
    height: 100%;
}

biochipman committed:
Фіксить стилі для мобільної верстки 414px

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -7,7 +7,7 @@ <h3 class="contacts__title">Ресторанно-готельний компле
+            <h4 class="mimino-address__title rest-title">Ресторан</h4>
             <a href="tel:+380955559533" class="mimino-address__link link">
               <svg class="mimino-address__icon" width="24" height="24">
                 <use href="/images/icon.svg#icon-phone"></use>
```
Постав крапку перед шляхом
./images/icon.svg#icon-phone

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -2,35 +2,29 @@
+              <a href="tel:+380668491089" class="mimino-address__link link">
                 <svg class="mimino-address__icon" width="24" height="24">
-                  <use href="/src/images/icon.svg#icon-phone"></use></svg
+                  <use href="/images/icon.svg#icon-phone"></use></svg
```
Постав крапку перед шляхом
./images/icon.svg#icon-phone

kuhai-anna commented on src/partials/main-page/contacts.html:
```diff
@@ -39,13 +33,13 @@ <h4 class="mimino-address__title">Готель</h4>
             >
               <svg class="mimino-address__icon" width="24" height="24">
-                <use href="/src/images/icon.svg#icon-point"></use>
+                <use href="/images/icon.svg#icon-point"></use>
```
Постав крапку перед шляхом
./images/icon.svg#icon-point

kuhai-anna commented on src/sass/layouts/main-page/_contacts.scss:
```diff
@@ -109,6 +111,18 @@
+
+.hotel-title {
+  @media screen and (min-width: 414px) and (max-width: 1023px) {
+    margin-right: 53px;
```
Постав тут марджин 54px, це я наплутала
